setup.rst

```diff
@@ -159,9 +247,13 @@ Go Deeper with Setup
 
 .. _`Stellar Development with Symfony`: http://symfonycasts.com/screencast/symfony
 .. _`Composer`: https://getcomposer.org/
-.. _`installed Composer`: https://getcomposer.org/download/
+.. _`Install Composer`: https://getcomposer.org/download/
+.. _`Install Symfony`: https://symfony.com/download
+.. _`install Symfony`: https://symfony.com/download
```

Comment:
duplicate line

Reply:
It looks like that ... but it has a different case: "install" vs "Install".

Reply:
I had not seen the different uses 😅